views/models/UsersModel/index.js

```diff
@@ -13,6 +13,24 @@ const UsersModel = {
       isFixed: reader === owner,
     })),
   createReadersList: (readers) => readers.map(({ value }) => value),
+  createUsersAccesList: (access) =>
+    access.map((site) => ({
+      label: site,
+      value: site,
+    })),
+  createUserFromFormValues: (user) => ({
```

Comment:
```suggestion
  userFromFormValues: (formValues) => ({
```

Reply:
I like the current function name but I think Mike is right about the argument variable name